### PR TITLE
Installation contributors

### PR DESCRIPTION
**Pull request by jamesgorrie**
# First pass
screen shot 2018-04-19 at 18 00 40

### CODE REVIEW

**Comment by jamesgorrie** [comment]
@wellcometrust/experience-dev taking this out of dev, I'll put another PR in with the UI stuff.

**Comment by jamesgorrie** [comment]
@wellcometrust/experience-dev scrap that comment... UI is basically done.
Waiting on a discussion with @jennpb and Jen before we make it better though.

File: common/model/contributors.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import type {HTMLString} from '../services/prismic/types';
 
 export type Organisation = {|
   name: string,
-  image: ?Picture,
+  image: Picture,
```
Comment (gestchild) [inline common/model/contributors.js]: is this really required?

Comment (jamesgorrie) [reply common/model/contributors.js]: Yup - if we don't get one from Prismic we will placehold it.
Thinking about getting [Proxies](vernacular) in for this.

**Pull request merged**